modules/EnsEMBL/Web/SpeciesDefs.pm

```diff
@@ -1095,6 +1050,84 @@ sub process_ini_files {
   }
 }
 
+
+sub _populate_taxonomy_division {
+# Populate taxonomy division using e_divisions.json template
+  my ($self, $tree, $url) = @_;
+
+  my $taxonomy = $tree->{$url}{'TAXONOMY'};
+  my $children = [];
+  my $other_species_children = [];
+
+  my @other_species = grep { $_->{key} =~ m/other_species/ } @{$tree->{'ENSEMBL_TAXONOMY_DIVISION'}->{child_nodes}};
+  $other_species[0]->{child_nodes} = [] if ($other_species[0] && !$other_species[0]->{child_nodes});
+
+  my $strain_name = $tree->{$url}{'SPECIES_STRAIN'};
+  my $strain_group = $tree->{$url}{'STRAIN_GROUP'};
+  my $group_name   = $tree->{$url}{'SPECIES_COMMON_NAME'};
+  my $species_key = $tree->{$url}{'SPECIES_URL'}; ## Key on actual URL, not production name
```

Comment:
we may have to change this to use production name. (I can't remember why I used species url) But this is a different issue that I will be working soon

Reply:
OK, I'll leave that one for now!